### ISSUE TITLE
Preliminary database research

### ISSUE THREAD

**Issue by kevincoleman** [kevincoleman/8bit#4]
Look into which database systems might be quick, accessible, and cross-platform for a game such as this.

Acceptance criteria: a list of 5 options for data storage, listing their strengths and weaknesses.

**Comment by nathan-isaac** [comment]
A document database might be interesting. For example [MongoDB](https://www.mongodb.com/).

**Comment by kevincoleman** [comment]
Yeah, I’ve tried MongoDB before, and it seemed ok. IIRC it’s unstructured—which may be ok at least for prototyping. Would we want better relational structures down the line?
